hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/TestPrefetch.java

```diff
@@ -298,6 +299,44 @@ public void testPrefetchDoesntSkipRefs() throws Exception {
     });
   }
 
+  @Test
+  public void testOnConfigurationChange() {
+    // change PREFETCH_DELAY_ENABLE_KEY from false to true
+    conf.setInt(PREFETCH_DELAY, 40000);
+    PrefetchExecutor.loadConfiguration(conf);
```

Comment:
Can't you use `onConfigurationChange` instead?

Reply:
Corrected